static/js/typing.js

```diff
@@ -92,6 +93,9 @@ export function get_recipient() {
         const stream_name = compose_state.stream_name();
         const stream_id = stream_data.get_stream_id(stream_name);
         const topic = compose_state.topic();
+        if (!stream_topic_history.get_recent_topic_names(stream_id).includes(topic)) {
+            return null; // Send notification only if topic already exists.
+        }
```

Comment:
I don't think this last change is correct or necessary -- one can narrow to a topic that doesn't exist, and if you do so at the same time as another user who is typing a message to that proposed new topic (E.g. someone said "Let's continue here" with a link), not adding this check will result in typing notifications working for those users.

(Well, we'd need to test the display UI in the empty narrow view for the topic, but that's it).

Reply:
Look like this in the empty narrow view: ![image](https://user-images.githubusercontent.com/44665669/113619636-db086580-9676-11eb-805c-4a7687b540a7.png)